Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -593,7 +620,8 @@ def samples_search_results_command():
         'Contents': results,
         'EntryContext': {'AutoFocus.SamplesResults(val.ID == obj.ID)': results,
```

Comment:
need to import linting here
build the entry context object before calling `demisto.results` and pass it here